service/controller/v19/adapter/guest_route_tables.go

```diff
@@ -32,7 +32,15 @@ func (r *GuestRouteTablesAdapter) Adapt(cfg Config) error {
 		TagName:      key.RouteTableName(cfg.CustomObject, suffixPublic),
 	}
 
-	for i := 0; i < key.SpecAvailabilityZones(cfg.CustomObject); i++ {
+	r.PrivateRouteTableNames = []RouteTableName{
+		{
+			ResourceName:        "PrivateRouteTable",
+			TagName:             key.RouteTableName(cfg.CustomObject, suffixPrivate),
+			VPCPeeringRouteName: "VPCPeeringRoute",
+		},
+	}
```

Comment:
My understanding so far was that with the new versioned clusters we use the new names, but in this PR we add the first item to the list like usual. So I would expect the first item to be removed for new multi AZ clusters in order to make them use the new naming and such. I feel like I miss why this here works.

Reply:
This isn't explained well I guess. I mentioned this alternative during our standup this morning. So the resources are named:

 * PrivateRouteTable
 * PrivateRouteTable01
 * PrivateRouteTable02

This way single AZ (old and new) have PrivateRouteTable and if we add an AZ they would get another one named `PrivateRouteTable01`. 

Benefit:
* No need to introduce new legacy clusters (that we might never get rid of anyway)
* No differences between old and new single AZ clusters

Drawback:
* omitting the PrivateRouteTable`00` will harm your feelings about consistency. but no real harm is done imo :ugly_duck:

Reply:
I understand. My feelings get hurt every day. Let's roll.